### ISSUE TITLE
Install on ubuntu (14.04?) fails on first attempt

### ISSUE THREAD

**Issue by museack** [mazaclub/tate#10]
I get the following error when trying to install Tate:
Error: Could not import icons_rc.py, please generate it with: 'pyrcc4 icons.qrc -o gui/qt/icons_rc.py'

If I run that command then rerun "python setup.py install" the installation works.

**Comment by guruvan** [comment]
@museack This is an issue electrum-wide - it's very unclear why the default setup.py doesn't run pyrcc4. Seems like it should. Do you know does this need to be run on a per system basis, or on a per OS type basis, or ???? it's a constant issue for electrum users

**Comment by museack** [comment]
It seems to be on a per-electrum basis.  Installed Tate had the issue.  Installed electrum-pot and it has the same problem.  I assume this is only an issue for users building from source.   I'm guessing Windows/OSX builds fail if you don't run this first.  (C:\program files (x86)\gui\qt/icons_rc.py and .pyc are present on my system)

The command "pyrcc4 icons.qrc -o gui/qt/icons_rc.py" needs to be run before the running "python setup.py install" or needs to be put in setup.py BEFORE files are put into the build dir.  

if it isn't done ahead of time, the icons_rc.py file is not present in tatesourcedir/gui/qt, this isn't copied to  the build directory (build/lib.linux-xxxxxx/tate_gui/qt) and hence doesn't get copied to /usr/local/lib/python2.7/dist-packages/Tate.version.egg/tate_gui/qt (or wherever it may be going).

I recommend just adding that command to the beginning of the setup.py file so it is run before anything else.  

I don't remember having this problem on Windows setup.exe version... presumably you'd already run that command before running the process to build the .exe

**Comment by guruvan** [comment]
Yup - it does depend on which forks have this fixed. It's definitely not clear why this isn't about the first thing in setup.py. 

And, yes, the binary builds include this command, or they would all be non functional. 

Will close when there's a PR to fix setup.py and setup-release.py (I do have some of this fixed in other forks)

**Comment by museack** [comment]
Added note to beginning of Readme to run the pyrcc4 command first!
